starlingcaptureapi/file_util.py

```diff
@@ -56,3 +58,51 @@ def digest_sha256(self, file_path):
                 hasher.update(byte_block)
             return hasher.hexdigest()
         # TODO: handle error (image not found, etc.)
+
+    def cidv1(self, file_path):
```

Comment:
Looks like this can be a `@staticmethod` (and possibly some other methods in this file could be static as well, but that's for a separate cleanup PR).

Reply:
I think we should name this `digest_cidv1` to be consistent with our `digest_sha256` category. Pls also add `digest_md5` based on last week's discussion. (Not necessarily in this PR.)

Reply:
Fixed in 3c7cf5b.

I created #71 to track the addition of MD5 hashing and assigned it to myself. If someone else should take it, or if it's a high priority then let me know.